source/documentation/other-topics/rotate-tokens-keys.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: Rotating tokens and keys
+last_reviewed_on: 2021-04-19
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+## Overview
+
+Below is the guidance for rotating the tokens and keys that are set as environment variables for your application container or your automated deployment pipeline. 
+This include only the variables fetched from system secrets from a namespace and not include any application related variables.
+
+### Serviceaccount tokens
+  Delete the secrets related to the serviceaccount credentials that are used in automated deployment pipeline like circleCI or github Actions. The secret name mostly start with `circleci-token-XXXX`
+
+    ```bash
+    $kubectl get secrets -n <yournamespace>
+    circleci-token-XXXXX                                       kubernetes.io/service-account-token   3      279d
+    default-token-jgtdz                                        kubernetes.io/service-account-token   3      279d
+    es-snapshot-bucket                                         Opaque                                2      144d
+    $
+    $kubectl delete secret circleci-token-XXXXX -n <yournamespace>
+
+  After the secret is deleted the replacement secret is generated immediately. Check the age of the newly generated secret to confirm. 
+  
+  Decode the secret using `cloud-platform decode-secret -n <yournamespace> -s circleci-token-XXXXX` to use it in your deployment pipeline.
+
+### AWS credentials
+   Contact `Cloud platform` team to rotate the credentials related to AWS resources such as ECR, RDS etc. The will run the instructions from the runbook [Rotate User AWS Credentials](https://runbooks.cloud-platform.service.justice.gov.uk/rotate-user-aws-credentials.html#rotate-user-aws-credentials)
```

Comment:
```suggestion
   [Ask the Cloud Platform team](/documentation/reference/getting-help.html) to rotate the credentials related to AWS resources such as ECR, RDS etc., which have been [created by terraform](/documentation/deploying-an-app/add-aws-resources.html). The team will run the instructions from the runbook [Rotate User AWS Credentials](https://runbooks.cloud-platform.service.justice.gov.uk/rotate-user-aws-credentials.html#rotate-user-aws-credentials) to regenerate the AWS credentials and they will copied into the existing kubernetes secret. This means that apps that [directly access the kubernetes secret](/documentation/deploying-an-app/add-secrets-to-deployment.html) can just be restarted to obtain the fresh value.
```

Reply:
I have updated the line slightly different. The pods dont need to be restarted to get fresh values. The secrets will be updated automatically in the mountPath. Let me know your thoughts.